src/admin-views/tribe-options-addons-api.php

```diff
@@ -5,6 +5,26 @@
  */
 $internal = array();
 
+$current_url = add_query_arg(
+	array(
+		'page' => 'tribe-common',
+		'tab' => 'addons',
+		'post_type' => Tribe__Events__Main::POSTTYPE,
+	),
+	admin_url( 'edit.php' )
+);
+
+if (
+	isset( $_GET['action'] )
+	&& isset( $_GET['_wpnonce'] )
+	&& 'disconnect-facebook' === $_GET['action']
+	&& wp_verify_nonce( $_GET['_wpnonce'], 'disconnect-facebook' )
+) {
+	tribe_update_option( 'fb_token', '' );
+	wp_redirect( $current_url );
+	die;
+}
```

Comment:
Can we move this to method outside of the `view`? Maybe put this on the `New.php` Tab where the Handling of the Authorization happens. 
Also, we need to delete: `fb_token_expires`and `fb_token_scopes`. Can set it to `null` too?